src/main/java/org/openrewrite/java/migrate/util/MigrateCollectionsSingletonMap.java

```diff
@@ -68,6 +68,13 @@ public J visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx)
 
                 return m;
             }
+
+            private boolean isNotLiteralNull(J.MethodInvocation m) {
+                return !(m.getArguments().get(0) instanceof J.Literal &&
+                         ((J.Literal) m.getArguments().get(0)).getValue() == null ||
+                         m.getArguments().get(1) instanceof J.Literal &&
+                         ((J.Literal) m.getArguments().get(1)).getValue() == null);
```

Comment:
I think we can make this slightly shorter and more expressive using
https://github.com/openrewrite/rewrite/blob/9abac8d9b5bd5bd4953569290b6591e91ed130cf/rewrite-java/src/main/java/org/openrewrite/java/tree/J.java#L3721-L3727

```suggestion
                return !J.Literal.isLiteralValue(m.getArguments().get(0), null) &&
                       !J.Literal.isLiteralValue(m.getArguments().get(1), null);
```